### ISSUE TITLE
read only camera should show recording duration (and schedule).

### ISSUE THREAD

**Issue by muneeb2542** [evercam/evercam-dashboard#1067]

**Comment by muneeb2542** [comment]
@marcoherbst  we do not allow a user to see the storage duration of the camera if he doesn't have edit rights.This is done straight from the cameras controller and we would have to change the structure to allow a read-only user to see recording duration.

**Comment by muneeb2542** [comment]
if you still want it,I can discuss with Sir Azhar on how to proceed on this with minimum drawback.

**Comment by marcoherbst** [comment]
@vinnieq, currently it's designed not to allow a read-only share to view the recordings settings. We could change this, but the decision was, initially, made deliberately.

This spreadsheet gives a full breakdown (this info was not on it, I have added it):
https://docs.google.com/a/evercam.io/spreadsheets/d/1SxhKjGvgaFgFZSmPLSpkggHtUEyQccYN9JnhIhF1aNo/edit?usp=sharing

My suggestion is that we leave it as it is. I made a note on the doc above to reconsider this during a makeover. Is that ok? I'll close this. Thanks Muneeb.
